app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.glia.exampleapp">
 
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
Is adding of this permission related to the scope of remote UI configuration?

Reply:
No, but it is impossible to enable notifications(for foreground service) without it on Android 12+, I added permission to testing app's manifest, maybe it makes sense to add this permission to widgets or  core sdk's manifest.